src/Microsoft.Health.Fhir.TemplateManagement/Utilities/OrasUtility.cs

```diff
@@ -0,0 +1,98 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.TemplateManagement.Exceptions;
+using Microsoft.Health.Fhir.TemplateManagement.Models;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.Utilities
+{
+    public static class OrasUtility
+    {
+        public static string GetOrasFileName()
+        {
+            string orasFileName;
+            switch (Environment.OSVersion.Platform)
```

Comment:
Maybe we can use [IsOSPlatform()](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.isosplatform?view=netcore-3.1) here?